DOCS-11052 category perms (#135)
mulesoft/docs-exchange: modules/ROOT/pages/to-manage-categories.adoc

endif::[]
:imagesdir: ../assets/images/

> Use categories to organize Exchange assets into groups and improve asset browsing and discovery.

Reviewer: Excellent rewording!
Is "and improve" more precise than "to improve"? Does the first cause the second, or are they separate things?

Author: The first causes the second, so I think "to improve" is more precise. Updated to use that.

For example, a category called *API Type* with possible values *System API*, *Experience API*,

> and *Integration API* is applied to assets of type *REST API*, *SOAP API*, and *HTTP API*:

Reviewer: Are these bold because they are displayed as labels in the image?
If not, unbold them.

Reviewer: Same for the final line below.

Author: Per our Slack discussion, these are not labels, so I removed the bold formatting.

image::ex-categories-api-type.png[]

> To create a category, you must have the role *Exchange Administrator* of the master organization or *Organization Owner* of the master organization.

Reviewer: Might be simpler to say...
```suggestion
To create a category, you must be either the master organization's Exchange Administrator or Organization Owner.
```

Author: Yes, I like that better. Updated.

When you have one or more categories defined in the *Settings* page, your organization's assets display the *Add Category* prompt.

> 

Reviewer: Use bold to quote UI labels that are right in front of the audience, that they can equate with what they see and act upon.

Author: Corrected.
